app/assets/javascripts/cvss.js

```diff
@@ -0,0 +1,13 @@
+(function() {
```

Comment:
Already declared in changelog.js so relying on that for the date picker and updated the id to work with that one.